packages/commonwealth/client/scripts/views/modals/edit_topic_thresholds_modal.tsx

```diff
@@ -46,16 +53,12 @@ const EditTopicThresholdsRow = (props: EditTopicThresholdsRowProps) => {
             e.preventDefault();
 
             try {
-              const status = await app.topics.setTopicThreshold(
+              await setTopicThreshold({
```

Comment:
Not sure if the catch statement below works?. [Related comment](https://github.com/hicommonwealth/commonwealth/pull/3945/files#r1201013784)

Reply:
I believe it should. That is why we use `mutateAsync` instead of `mutate`. Ideally, I agree that we should use `error, loading and data` from the react-hook but it is different mental model than we have right now in the app. IMO it is easier to transfer with the approach I took but for new components or refactors, we should go the way you are proposing so destructuring error, data and loading

Reply:
Sounds good!.